scandinavian-journal-of-rheumatology.csl

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" default-locale="en-GB" page-range-format="minimal" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Scandinavian Journal of Rheumatology</title>
+    <id>http://www.zotero.org/styles/scandinavian-journal-of-rheumatology</id>
+    <link href="http://www.zotero.org/styles/scandinavian-journal-of-rheumatology" rel="self"/>
+    <link href="http://www.zotero.org/styles/annals-of-oncology" rel="template"/>
+    <link href="http://www.tandf.co.uk/journals/authors/healthcare/IRHE.pdf" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien, PhD</name>
+      <email>obrienpat86@gmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0300-9742</issn>
+    <eissn>1502-7732</eissn>
+    <updated>2016-12-26T20:54:55+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editor" form="short">
+        <single>ed</single>
+        <multiple>eds</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor" prefix=" In: " suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always" et-al-min="5" et-al-use-first="3" delimiter-precedes-et-al="never"/>
+      <label form="short" prefix=" "/>
+      <substitute>
+        <names variable="editor"/>
+      </substitute>
```

Comment:
@damnation333, I see that this same code exists in https://github.com/citation-style-language/styles/blob/master/annals-of-oncology.csl, but can you still fix it here? Just take out the whole `<substitute/>` (lines 37-39). It makes no sense to substitute the "editor" variable in case of a missing "editor" variable.